### PR TITLE
EHealth-3448: added validation logic for reimbursement contract request update

### PR DESCRIPTION
**Pull request by PavelVesnin**
#3448

### CODE REVIEW

File: apps/ehealth/test/web/controllers/contract_request/capitation_controller_test.exs [hunk 1]
```diff
@@ -1232,20 +1225,14 @@ defmodule EHealth.Web.ContractRequest.CapitationControllerTest do
...
-
-      schema =
-        "../core/specs/json_schemas/contract_request/capitation_contract_request_show_response.json"
-        |> File.read!()
-        |> Jason.decode!()
-
-      assert :ok = NExJsonSchema.Validator.validate(schema, resp["data"])
+      conn
+      |> post(contract_request_path(conn, :create, @capitation, UUID.generate()), %{
+        "signed_content" => params |> Jason.encode!() |> Base.encode64(),
```
Comment (progsmile) [inline apps/ehealth/test/web/controllers/contract_request/capitation_controller_test.exs]: there was a function somewhere that wraps request data with signed_content map, suggest to use it

File: apps/graphql/test/web/resolvers/reimbursement_contract_request_resolver_test.exs [hunk 2]
```diff
@@ -101,6 +115,127 @@ defmodule GraphQLWeb.ReimbursementContractRequestResolverTest do
...
+        }
+      }
+
+      resp_body =
+        conn
+        |> put_client_id(legal_entity.id)
+        |> post_query(@update_query, variables)
+        |> json_response(200)
+
+      refute resp_body["errors"]
```
Comment (progsmile) [inline apps/graphql/test/web/resolvers/reimbursement_contract_request_resolver_test.exs]: let's move `refute` below `resp_entity`

File: apps/ehealth/test/web/controllers/contract_request/capitation_controller_test.exs (hunk 1 above)
Comment (progsmile) [reply apps/ehealth/test/web/controllers/contract_request/capitation_controller_test.exs]: it's `input_signed_content` on graphql

Comment (PavelVesnin) [reply apps/ehealth/test/web/controllers/contract_request/capitation_controller_test.exs]: It's EHealth application tests. Every test post signed content in the same way.
I think that your suggestion for separate PR

File: apps/graphql/test/web/resolvers/reimbursement_contract_request_resolver_test.exs (hunk 2 above)
Comment (PavelVesnin) [reply apps/graphql/test/web/resolvers/reimbursement_contract_request_resolver_test.exs]: what's the idea?

Comment (progsmile) [reply apps/graphql/test/web/resolvers/reimbursement_contract_request_resolver_test.exs]: separate assignments from assertions

**Pull request merged**